osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -30,6 +30,11 @@ public class OsuDifficultyHitObject : DifficultyHitObject
         /// </summary>
         public readonly double StrainTime;
 
+        /// <summary>
+        /// Milliseconds elapsed since the start time of the previous <see cref="OsuDifficultyHitObject"/>, with a minimum of 25ms, adjusted with possible cheese.
+        /// </summary>
+        public double AdjustedStrainTime;
```

Comment:
I'm not sure this even needs to exist - you do all of the doubletap adjustments in aim evaluator and storing the adjusted strain isn't useful

Reply:
i can move it to EvaluateDifficulty func